plugins/content_distribution/admin/forms/ProviderProfileConfiguration.php

```diff
@@ -101,7 +103,20 @@ public function init()
 		$this->addProfileAction('delete');
 		$this->addProfileAction('report');
 	}
-	
+
+	protected function addDistributeOnTypeElement()
+	{
+		$this->addElement('select', "distribute_On_Type", array(
+			'label'	  =>  'Distribute On',
+			'value'			=> $this->distributionProfile->distributeOnType,
+			'decorators' => array('ViewHelper', array('Label', array('placement' => 'prepend')), array('HtmlTag',  array('tag' => 'dt')))
+		));
+
+		$element = $this->getElement("distribute_On_Type");
+		$element->addMultiOption(Kaltura_Client_ContentDistribution_Enum_DistributeOnType::ENTRY_READY, 'On entry ready');
```

Comment:
Instead of adding multi-option you need to add Kaltura_Form_Element_EnumSelect and add the enum translation in the en.php (to admin console and the var console).
This is the more correct way to do this.